modules/updating-eus-to-eus-layered-products.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-olm-operators_{context}"]
+= EUS-to-EUS update for layered products and Operators installed through Operator Lifecycle Manager (OLM) 
+
+In addition to the EUS-to-EUS update steps mentioned for the web console and CLI, there are a couple of additional steps to consider when performing EUS-to-EUS updates for the clusters of layered products and those of Operators that have been installed through the Operator Lifecycle Manager (OLM). 
+
+.What is a layered product?
+
+Layered products refer to products that are made of multiple underlying products that are intended to be used together and cannot be broken into individual subscriptions. For examples of layered {product-title} products, see link:https://access.redhat.com/support/policy/updates/openshift/#layered[Layered Offering On OpenShift].
+
+As you perform an EUS-to-EUS update for the clusters of layered products and those of Operators that have been installed through OLM, you must complete the following:
+
+. Ensure that all of your Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures that they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update. For information on how to update your Operators, see _Preparing for an Operator update_ in Additional resources.
+
+. Confirm the API and kernel compatibility used in the current and intended Operator versions. You can verify which versions your OLM operators are compatible with by using the link:https://access.redhat.com/labs/ocpouic/?operator=logging&&ocp_versions=4.10,4.11,4.12[Red Hat {product-title} Operator Update Information Checker].
```

Comment:
What's the origin of this kernel compatibility concern? Maybe this should just be "Confirm the cluster version compatibility..." ?

Reply:
This came from the ticket as a concern, but if the user can just confirm the cluster version compatibility, we should we good :)